website/content/docs/release-notes/nomad/upcoming.mdx

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+page_title: Upcoming
+description: >-
+  Notices of upcoming Nomad changes
+---
+
+# Upcoming
+
+Nomad strives to be backward compatible for at least 2 major releases, so for
+example Nomad v1.6.x can be upgraded to Nomad v1.8.x. In addition, Enterprise
+customers have access to [Long Term Support (LTS)][LTS] releases.
+
+Refer to [Upgrading][] for details of the upgrade process, and [Upgrade
+Guides][] for notes on upgrading specific versions.
+
+However, from time to time it's necessary to make backwards incompatible changes
+to Nomad. These are announced on [changelog][], the [Release Notes][], and the
+version-specific [Upgrade Guides][]. In order to help users plan their upgrades,
+this page will maintain a list of all planned future changes that might cause
+backwards incompatibilities or require operational changes. It's possible that a
+given change may happen later than intended, at which point this document will
+be updated. But changes will not be moved earlier than originally published
+unless there's a required change due to a security vulnerability.
```

Comment:
```suggestion
However, from time to time it's necessary to make backwards _incompatible_ changes
to Nomad. We announce these on [changelog][], the [Release Notes][], and the
version-specific [Upgrade Guides][]. In order to help you plan your upgrades,
this page maintains a list of all planned future changes that might cause
backwards incompatibilities or require operational changes. It's possible that a
given change may happen later than intended, at which point we update this document. However, we do not move changes earlier than originally published
unless there's a required change due to a security vulnerability.
```

some style guide nits. 
I also emphasized "incompatible" to make it stand out since my brain read it as "compatible" the first time I scanned the paragraph.